<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>711</height>
     </rect>
    </property>
    <widget class="QStatusBar" name="statusbar_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>3</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>361</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortDesc">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировать ↓</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortAsc">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировать ↑</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>31</y>
       <width>311</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Введите размер массива</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>631</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>594</y>
       <width>631</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>105</y>
       <width>371</width>
       <height>192</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>124</number>
     </attribute>
     <column>
      <property name="text">
       <string>Номер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>392</x>
       <y>105</y>
       <width>321</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>2</width>
       <height>2</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="comboListLayout"/>
    </widget>
    <widget class="QPushButton" name="btnAddList">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>513</y>
       <width>147</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить в список</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveRow">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>513</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить строку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveList">
     <property name="geometry">
      <rect>
       <x>443</x>
       <y>513</y>
       <width>143</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить из списка</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddRow">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>513</y>
       <width>132</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить строку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortListCustom">
     <property name="geometry">
      <rect>
       <x>593</x>
       <y>513</y>
       <width>158</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировать список</string>
     </property>
    </widget>
   </widget>
   <widget class="QStatusBar" name="statusbar_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>3</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
